packages/gasket-plugin-cypress/test/index.test.js

```diff
@@ -0,0 +1,152 @@
+    expect(plugin).toBeInstanceOf(Object);
+  });
+
+  it('has expected name', function () {
+    expect(plugin).toHaveProperty('name', require('../package').name);
+  });
+
+  it('has expected hooks', function () {
+    const expected = ['create', 'metadata'];
+
+    expect(plugin).toHaveProperty('hooks');
+
+    const hooks = Object.keys(plugin.hooks);
+    expect(hooks).toEqual(expected);
+    expect(hooks).toHaveLength(expected.length);
+  });
+
+  it('has the correct create hook timings', function () {
+    expect(plugin.hooks.create.timing.after).toEqual(['@gasket/plugin-nextjs']);
+  });
+
+  it('has the correct custom cypress config', async function () {
+    expect(config).toHaveProperty('baseUrl', 'http://localhost:8080');
+    expect(config).toHaveProperty('integrationFolder', 'test');
+    expect(config).toHaveProperty('video', false);
+  });
+
+  describe('react', function () {
+    it('includes a glob for the `generator/cypress.json` contents for react projects', async function () {
+      const { files } = await createReact();
+
+      expect(files[0]).toContain('/../generator/*');
+      expect(files[1]).toContain('/../generator/**/*');
+    });
+  });
+
+  describe('adds react specific dependencies', function () {
+    ['cypress', 'start-server-and-test'].forEach((name) => {
+      it(`adds "${name}" in the devDependencies`, async function () {
+        const { pkg } = await createReact();
+
+        expect(pkg.devDependencies).toHaveProperty(name);
+      });
+    });
+
+    it('depends on the same versions', async function () {
+      const { pkg } = await createReact();
+
+      expect(typeof pkg.devDependencies).toBe('object');
+      Object.keys(pkg.devDependencies).forEach((key) => {
+        expect(self.devDependencies).toHaveProperty(key);
+        expect(self.devDependencies[key]).toEqual(pkg.devDependencies[key]);
+      });
+    });
+  });
+
+  describe('dependencies', function () {
+    it('adds "cypress" in the devDependencies', async function () {
+      const { pkg } = await create();
+
+      expect(pkg.devDependencies).toHaveProperty('cypress');
+    });
+
+    it('depends on the same versions', async function () {
+      const { pkg } = await create();
+
+      expect(typeof pkg.devDependencies).toBe('object');
+      Object.keys(pkg.devDependencies).forEach((key) => {
+        expect(self.devDependencies).toHaveProperty(key);
+        expect(self.devDependencies[key]).toEqual(pkg.devDependencies[key]);
+      });
+    });
+  });
+
+  describe('scripts', function () {
+    it('uses the same scrips in our package.json', async function () {
+      const { pkg } = await create();
+
+      self.scripts = {
+        ...self.scripts,
+        'cypress': 'cypress open',
+        'cypress:headless': 'cypress run',
+        'e2e': 'start-server-and-test start:local http://localhost:8080 cypress',
+        'e2e:headless':
+          'start-server-and-test start:local http://localhost:8080 cypress:headless'
+      };
```

Comment:
The add scripts do not need to be a part of plugin's package. Just assign them to an `expected` object for your checks below.

```suggestion
      const expected = {
        'start:local': 'gasket start --env local',
        'cypress': 'cypress open',
        'cypress:headless': 'cypress run',
        'e2e': 'start-server-and-test start:local http://localhost:8080 cypress',
        'e2e:headless':
          'start-server-and-test start:local http://localhost:8080 cypress:headless'
      };
```